firefox-ios/Client/Application/RemoteSettings/RemoteRecords/PasswordRuleRecord.swift

```diff
@@ -0,0 +1,28 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+protocol RemoteDataTypeRecord: Codable, Equatable {}
+
+struct PasswordRuleRecord: RemoteDataTypeRecord {
+    let domain: String
+    let passwordRules: String
+    let id: String
+    let lastModified: Int
```

Comment:
Very minor nit: is it possible to comment/document what the Int/timestamp represents (seconds since epoch etc.)?

Reply:
Good call, it is indeed unix time aka epoch timestamp (I added a comment)